package.json

```diff
@@ -13,11 +13,7 @@
     "postinstall": "husky install"
   },
   "lint-staged": {
-    "**/*.{js,jsx}": "prettier --write",
-    "**/*.{ts,tsx}": [
-      "prettier --write",
-      "tsc-files --noEmit"
```

Comment:
I removed the TS compile check from the pre-commit hook. It wasn't working correctly. Spent a little time to try and fix it, but couldn't figure it out and decided to punt on it.